models/validateEntity.go

```diff
@@ -339,18 +364,25 @@ func ValidateEntity(entity int, t map[string]interface{}) (map[string]interface{
 	// Extra checks
 	switch entity {
 	case u.SITE, u.BLDG, u.ROOM, u.RACK, u.DEVICE, u.AC,
-		u.PWRPNL, u.CABINET, u.CORIDOR, u.SENSOR, u.GROUP:
+		u.PWRPNL, u.CABINET, u.CORRIDOR, u.SENSOR, u.GROUP:
+		if strings.ContainsAny(t["name"].(string), ".") {
```

Comment:
Can we move the name validation from API -> refs/base.json? I might also to forbid whitespaces in the names too